website/docs/examples/next-rust.md

```diff
@@ -1,15 +0,0 @@
----
-sidebar_position: 4
----
-
-# Next with Next.js compiler (Rust compiler)
-
-Available for Next.js version `12.0.0` or above.
-
-## Integrate Jest and Jest Preview to your Next.js app
-
-See more at [Next.js Example](https://github.com/nvh95/jest-preview/tree/main/examples/nextjs)
-
-## Caveats
-
-Next's `<Image />` component has yet to work with Jest Preview.
```

Comment:
This is not mentioned in https://github.com/nvh95/jest-preview/tree/main/examples/nextjs
Please help to move it there.

Reply:
Actually they are there 

```
🚧 Next.js's <Image /> component isn't working with Jest Preview yet.
```